generators/app/templates/src/containers/TicTacToe/GameListPage/children/List/children/GameItem/game-item.test.js

```diff
@@ -4,9 +4,9 @@ import { render } from 'react-testing-library';
 import GameItem from './game-item.component';
 
 const game = {
-  gamestatus: 'finished',
+  status: 'finished',
```

Comment:
I just noticed elsewhere it's checking for "Finished" instead of "finished" - should this be capitalized?

Reply:
This is for compare doesn't show to the user, for the user we are using capitalized version.